x-pack/plugins/triggers_actions_ui/public/application/lib/action_connector_api.ts

```diff
@@ -11,7 +11,7 @@ import type { ActionConnector, ActionConnectorWithoutId, ActionType } from '../.
 import { ActionTypeExecutorResult } from '../../../../../plugins/actions/common';
 
 export async function loadActionTypes({ http }: { http: HttpSetup }): Promise<ActionType[]> {
-  return await http.get(`${BASE_ACTION_API_PATH}/list_action_types`);
+  return await http.get(`${BASE_ACTION_API_PATH}/list_connector_types`);
```

Comment:
I think the API is `/connector_types`.

```suggestion
  return await http.get(`${BASE_ACTION_API_PATH}/connector_types`);
```